flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why excluding aarch64?

Reply:
Currently due to github actions limitations the app was only built for x86, although I am looking for any flathub app that better be built with flutter using build script.